paying_for_college/templates/worksheet.html

```diff
@@ -150,32 +147,40 @@ <h2 class="verify_prompt">
+                                tool, you’ll:
                             </h2>
-                            <p>
-                                In <strong>Step 1</strong>, review your
-                                first-year financial aid offer and update any
-                                incorrect information. This helps us provide a
-                                more accurate evaluation.
-                            </p>
-                            <p>
-                                In <strong>Step 2</strong>, evaluate your
-                                offer to learn about the risks and rewards of
-                                accepting it. We include things like
-                                graduation rate, affordability, and loan
-                                default rates.
-                            </p>
-                            <p>
-                                In <strong>Step 3</strong>, learn about your
-                                options when it comes to how to reduce student
-                                debt for a more positive financial future.
-                            </p>
+                            <ol>
+                                <li>Review your first-year
+                                financial aid offer and update any incorrect
+                                information.</li>
+                                <li>Evaluate the risks and
+                                rewards of accepting your financial aid offer
+                                by reviewing graduation rate, affordability,
+                                loan default rates, and other factors specific
+                                to your school and program.</li>
+                                <li>Learn about your options
+                                to reduce student debt.</li>
+                            </ol>
                             <p class="instructions_subheading">
                                 You must review all three parts of this tool
                                 before you can enroll.
                             </p>
                         </div>
+                        <div class="instructions_about-wrapper">
+                            <h5 class="instructions_about-heading">
+                                About this tool
+                            </h5>
+                            <div class="instructions_about">
+                                <p>This tool uses data and financial terms that
+                                    may need additional explanation. Learn more
+                                    about the financial language, how
+                                    calculations are created, and where the
+                                    data comes from.
+                                </p>
+                                <a href="about-this-tool/">Get details about how this tool works</a>
```

Comment:
This relative link won't work. the 'about' page does not have `/offer/` in its path.
needs to go up a level to `/understanding-your-financial-aid-offer/about-this-tool/`